src/library_browser.js

```diff
@@ -225,6 +225,33 @@ var LibraryBrowser = {
       };
       Module['preloadPlugins'].push(audioPlugin);
 
+#if (WASM != 0) && (MAIN_MODULE != 0)
```

Comment:
I don't believe this works. Do you see it prevent the code from being included when it should not be?

All I am sure works is
```
#if WASM
#if MAIN_MODULE
...
#endif
#endif
```
Sorry about the ugliness, we should improve the preprocessor (which is literally just a few lines of JS at this point)